tests/docker_test.go

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (C) 2015 Red Hat, Inc.
```

Comment:
year?

Reply:
Those tests were in topology_test.go, I kept the original date

Reply:
ok, but the file creation date will be new right? is that fine? if yes no problem for me.